rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/custom_service_info.hpp

```diff
@@ -62,6 +71,91 @@ typedef struct CustomServiceRequest
   : buffer_(nullptr) {}
 } CustomServiceRequest;
 
+class ServicePubListener : public eprosima::fastrtps::PublisherListener
+{
+  using subscriptions_set_t =
+    std::unordered_set<eprosima::fastrtps::rtps::GUID_t,
+      rmw_fastrtps_shared_cpp::hash_fastrtps_guid>;
+  using clients_endpoints_map_t =
+    std::unordered_map<eprosima::fastrtps::rtps::GUID_t,
+      eprosima::fastrtps::rtps::GUID_t,
+      rmw_fastrtps_shared_cpp::hash_fastrtps_guid>;
+
+public:
+  explicit ServicePubListener(CustomServiceInfo * info)
+  : info_(info)
```

Comment:
Maybe I'm missing something, but it doesn't look like `info_` is used anywhere. Do we need to make this API change or can we revert it?

Reply:
Thanks @jacobperron!!

It is true! This change was introduced by a previous solution I tried and I forgot to revert it. It is no longer needed and I have reverted it in the last commit. I will create another pull request to revert it in master too.